frontend/packages/core/src/Table/metadata-table.tsx

```diff
@@ -76,12 +76,19 @@ const TableCell = styled(MuiTableCell)({
 });
 
 const Grid = styled(MuiGrid)({
+  display: "flex",
   ".MuiFormControl-root": {
+    flexDirection: "row",
+  },
+  ".MuiFormControl-root .MuiInputBase-root": {
     height: "40px",
     width: "100px",
-    flexDirection: "row",
+    alignSelf: "center",
```

Comment:
so solve for this alignment issue with longer errors

<img width="379" alt="Screen Shot 2021-01-06 at 5 02 15 PM" src="https://user-images.githubusercontent.com/39421794/103823906-f4325380-5040-11eb-9cc9-e51b4d62b730.png">